clients/ui/frontend/src/app/pages/modelCatalog/screens/RegisterCatalogModelPage.tsx

```diff
@@ -32,10 +32,10 @@ const RegisterCatalogModelPageInner: React.FC = () => {
   const preferredModelRegistry = modelRegistries.length > 0 ? modelRegistries[0] : null;
 
   // Check to see if data is loaded
+  // Note: We allow the form to render even if artifacts fail to load - the form will handle empty artifacts gracefully
   const isDataReady =
     modelLoaded &&
     artifactLoaded &&
-    !artifactsLoadError &&
```

Comment:
we should keep this. I think the error we are facing is not from this
but from the details page we show uri field - the page is breaking because of that. we need to fix this in `ModelDetailsView`